File: ai_agent_3/src/main/java/com/example/invoiceagent/InvoiceAgentApplication.java
package com.example.invoiceagent;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan
public class InvoiceAgentApplication {

    public static void main(String[] args) {
        SpringApplication.run(InvoiceAgentApplication.class, args);
    }
}

File: invoice-agent/src/main/kotlin/com/agent/liquidalts/invoiceagent/InvoiceAgentApplication.kt
package com.agent.liquidalts.invoiceagent

import com.agent.liquidalts.invoiceagent.config.InvoiceAgentProperties
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableConfigurationProperties(InvoiceAgentProperties::class)
class InvoiceAgentApplication

fun main(args: Array<String>) {
    runApplication<InvoiceAgentApplication>(*args)
}
